src/stackwalk.c

```diff
@@ -139,11 +139,16 @@ JL_DLLEXPORT jl_value_t *jl_backtrace_from_here(int returnsp)
     return bt;
 }
 
-JL_DLLEXPORT void jl_get_backtrace(jl_array_t **btout, jl_array_t **bt2out)
+JL_DLLEXPORT void jl_get_backtrace(jl_value_t **btout, jl_value_t **bt2out)
 {
     jl_ptls_t ptls = jl_get_ptls_states();
     jl_array_t *bt = NULL;
     jl_array_t *bt2 = NULL;
+    if (ptls->julia_bt != NULL) {
+        *btout = ptls->julia_bt;
+        *bt2out = jl_nothing;
```

Comment:
what is `bt2` again?

Reply:
`MethodInstance`s for interpreted frames.

Reply:
Can we fix the name and document what that means?